internal/tools/tools.go

```diff
@@ -26,25 +26,32 @@ package tools
 import (
 	// thriftrw code gen
 	_ "go.uber.org/thriftrw"
+
 	// yarpc plugin for thriftrw code gen
 	_ "go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-yarpc"
 
 	// protobuf stuff
 	_ "github.com/gogo/protobuf/protoc-gen-gofast"
 	_ "go.uber.org/yarpc/encoding/protobuf/protoc-gen-yarpc-go"
 
-	// goimports
-	_ "golang.org/x/tools/cmd/goimports"
+	// forces one import-order pattern
+	_ "github.com/NonLogicalDev/gofancyimports/cmd/gofancyimports"
+
```

Comment:
I'm slightly inclined to make these trailing so they sort together...

... but also meh.  happy to do so if someone feels more strongly.

Reply:
no need in gci, though trailing might be a bit nicer to read